pkg/startup/v6/data/master/etc/origin/node/node-config.yaml

```diff
@@ -33,6 +33,16 @@ kubeletArguments:
   - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true
   max-pods:
   - "50"
+  image-gc-high-threshold: 
+  - "80"
+  image-gc-low-threshold: 
```

Comment:
This is the one im interested in. Default is 80. And I remember we always were setting this lower than that as 80 sometimes is too big when you are on 79% and somebody tries to deploy "big fat daemon".